CHANGELOG.md

```diff
@@ -10,16 +10,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - [2079](https://github.com/FuelLabs/fuel-core/pull/2079): Open unknown columns in the RocksDB for forward compatibility.
 
 ### Changed
-- [2076](https://github.com/FuelLabs/fuel-core/pull/2076): Replace usages of `iter_all` with `iter_all_keys` where necessary.
 
 #### Breaking
 
+- [2080](https://github.com/FuelLabs/fuel-core/pull/2080): Reject Upgrade txs with invalid wasm on txpool level.
+- [2076](https://github.com/FuelLabs/fuel-core/pull/2076): Replace usages of `iter_all` with `iter_all_keys` where necessary.
```

Comment:
It is non breaking change, so it should be between `Change` and `Breaking`=)

Reply:
I did remove one unused variant of the error enum, which I'd assume makes this change count as breaking.

Reply:
Your change is breaking, but the change 2076 is non-breaking=)

Reply:
Oh I see, yeah this must be a rebase artifact. I'll fix.

Reply:
Done ✅